posts/2021-11-30-microprofile-5-22001-beta.adoc

```diff
@@ -0,0 +1,203 @@
+
+To use this new feature, you would need to add `mpRestClient-3.0` to the `featureManager` element in the server.xml. The code should be similar to previous versions of MP Rest Client, but the packages should change from `javax.*` to `jakarta.*`.
+
+[source, xml]
+----
+<featureManager>              
+    <feature>mpRestClient-3.0</feature>
+</featureManager>
+----
+
+For more information, check out:
+
+* link:https://github.com/eclipse/microprofile-rest-client[Eclipse MicroProfile Rest Client GitHub]
+* link:http://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC5/microprofile-rest-client-spec-3.0-RC5.html[MicroProfile Rest Client 3.0 Specification]
+* link:http://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC5/apidocs/[MicroProfile Rest Client 3.0 API Javadoc]
+
+
+[#jpa_persistence]
+=== Define JPA persistence properties at server scope
+
+This new JPA enhancement adds the ability to declare "default" JPA persistence properties, to all container-managed persistence contexts, as a Liberty server.xml configuration.
+
+Previously, if a persistence property needed to be set for all persistence.xml configuration files, you would manually update all persistence.xml files in all applications. This could end up requiring hundreds of manual updates and/or rebuilding of applications.
+With this enhancement, you can specify persistence properties, in the server.xml, that will propagate to all container-managed persistence units for applications installed on that server.
+
+In order to start using the new feature, you only need to add the new `<defaultProperties>` configuration to your `<jpa>` configuration, in server.xml, along with the persistence properties that you want to apply to all container-managed persistence units:
+
+===== Example 1:
+
+[source, xml]
+----
+<server>
+    <featureManager>
+        <feature>persistenceContainer-3.0</feature>
+    </featureManager>
+
+    <jpa defaultPersistenceProvider="org.hibernate.jpa.HibernatePersistenceProvider">
+        <defaultProperties>
+            <property name="javax.persistence.lock.timeout" value="4000"/>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.Oracle12cDialect"/>
+        </defaultProperties>
+    </jpa>
+</server>
+----
+
+===== Example 2:
+
+[source, xml]
+----
+<server>
+    <featureManager>
+        <feature>jpa-2.1</feature>
+    </featureManager>
+
+    <jpa>
+        <defaultProperties>
+            <property name="javax.persistence.lock.timeout" value="12345"/>
+            <property name="eclipselink.cache.shared.default" value="false"/>
+        </defaultProperties>
+    </jpa>
+</server>
+----
+
+==== Technical description
+
+These `defaultProperties`, integration-level persistence properties will be supplied to the persistence provider specified when the `PersistenceProvider.createContainerEntityManagerFactory` method is called by the JPA Container.
```

Comment:
These `defaultProperties` are integration-level persistence properties that are supplied to the specified persistence provider when the `PersistenceProvider.createContainerEntityManagerFactory` method is called by the JPA Container